unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
@dhylands: thanks for the explanation, I think I'm beginning to understand the magic behind the qstr's. Please correct me if I'm wrong, but as far as I can tell there is one "master" /py/qstrdefs.h and then, for each port, there *could* be a specific qstr definition that gets added in addition to the *master* qstr, since they just get processed in series as far as I can see in the props file: `<Exec Command="python $(PySrcDir)makeqstrdata.py $(PreProc) $(QstrDefs) > $(TmpFile)"/>` and the `do_work` function in `/py/makeqstrdata.py`. It doesn't seem, however, that there is a way to be able to concatenate more than two of those qstr headers (at least for the Windows project), so adding lib-specific qstr's would require hand-crafting the port-specific one.

In any case, it seems to me now like `Q(settimeout)`, being something needed for lwIP and not the Windows/unix port, **definitely** does not belong to the `/unix/qstrdefsport.h`. Would that be correct?